src/main/java/org/jboss/logmanager/handlers/SyslogHandler.java

```diff
@@ -780,14 +779,11 @@ public void setEscapeEnabled(final boolean escapeEnabled) {
      * Returns the pid being used as the PROCID for RFC5424 messages.
      *
      * @return the pid or {@code null} if the pid could not be determined
+     * @deprecated Always returns the current process ID, as a string.
      */
+    @Deprecated(forRemoval = true, since = "3.1")
```

Comment:
3.1 seems okay here, but we will need to migrate to 3.1 as current next release is 3.0.5.

Reply:
Yeah it's an API change so I figured it warranted a minor version bump. I can update the pom as well.